magenta/interfaces/midi/midi.py

```diff
@@ -198,11 +209,13 @@ class Metronome(threading.Thread):
   """
   daemon = True
 
-  def __init__(self, outport, bpm, clock_start_time):
+  def __init__(self, outport, bpm, clock_start_time, velocity, channel):
     self._outport = outport
     self._bpm = bpm
     self._stop_metronome = False
     self._clock_start_time = clock_start_time
+    self._metronome_velocity = int(round(velocity))
```

Comment:
Let's just call these _velocity and _channel for simplicity since it's clear they relate to the metronome.